packages/viewer/src/components/viewer-annotation-callout/viewer-annotation-callout.css

```diff
@@ -46,3 +48,8 @@
 .icon {
   color: var(--viewer-annotation-callout-accent-color);
 }
+
+.occluded {
+  opacity: .3;
+  transition: opacity 0.75s ease-in;
+}
```

Comment:
There's no way for an application to customize these styles. Let's remove this and let the application (Connect) provide their styles for occlusion. This has the added benefit of not having to make changes to the SDK if we want to make styling changes in Connect. Related https://github.com/Vertexvis/vertex-web-sdk/pull/597/files/58a5dcd89baab5b59f5cc6df6d09b563d2c19af7#r1588393719

Reply:
I removed the styling here.